.github/workflows/ci.yml

```
@@ -48,6 +57,108 @@ jobs:
         path: output
 
     - name: Upload to Codecov
-      uses: codecov/codecov-action@v2
-      with:
-        token: ${{ secrets.CODECOV_TOKEN }}
+      uses: codecov/codecov-action@v3
+
+  build-taco:
+    env:
+#      SIGNING_ENABLED: ${{ github.event.inputs.signTacoFile }}
+      SIGNING_ENABLED: true
```

```suggestion
      SIGNING_ENABLED: ${{ github.event.inputs.signTacoFile }}
```
We can add the flag back now that the signing worked